crates/bsp_types/src/requests/cargo_extension/cargo_features_state.rs

```diff
@@ -0,0 +1,142 @@
+use super::Feature;
+use crate::BuildTargetIdentifier;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeSet;
+
+use crate::requests::Request;
+
+#[derive(Debug)]
+pub enum CargoFeaturesState {}
+
+impl Request for CargoFeaturesState {
+    type Params = ();
+    type Result = CargoFeaturesStateResult;
+    const METHOD: &'static str = "workspace/CargoFeaturesState";
```

Comment:
```suggestion
    const METHOD: &'static str = "workspace/cargoFeaturesState";
```
konwencja jest żeby nazwy requestów były camelCasem pisane